### ISSUE TITLE
downloads section. Stop changing vid order. 

### ISSUE THREAD

**Issue by steviefaux** [futo-org/grayjay-android#1611]
### Use Cases

Maybe I'm missing something but every time I go into the download section the order of all downloaded videos has changed. Even if I haven't downloaded a new vid. It seems to change the order every time.

### Proposal

Can we get a sort order in downloads or at least stop it randomly sorting the vids every time you go into downloads.

**Comment by Uddhav07** [comment]
Also the downloads button is missing in yt videos

**Comment by Zvonimir-FUTO** [comment]
> Also the downloads button is missing in yt videos

Are you using play store version? If so that is expected since Google removed Grayjay from Play Store for having an option to download and play audio in background.

**Comment by Uddhav07** [comment]
> > Also the downloads button is missing in yt videos
> 
> Are you using play store version? If so that is expected since Google removed Grayjay from Play Store for having an option to download and play audio in background.

would it be possible to get non play store version from galaxy store? {quite easy and less restrictive}
i dont wanna install from unkown sources, i know it must be safe but i follow a simple rule for my main phone

**Comment by Zvonimir-FUTO** [comment]
You can also download it from F-droid as well. No plans for Galaxy store yet.

**Comment by Uddhav07** [comment]
> You can also download it from F-droid as well. No plans for Galaxy store yet.

Can we have exporting YouTube downloaded playlist like we can for individual videos

**Comment by Zvonimir-FUTO** [comment]
Think there is already a feature request for that. We will probably add it in future although it is not a priority.

**Comment by steviefaux** [comment]
Might not be a priority but the random sorting everytime you go into downloads I'd class as a unforseen bug and is super annoying.

**Comment by Uddhav07** [comment]
yeah my some videos show downloaded in the playlist section but not in the playlist doesnt show up on the downloads tab
